src/tests/suite/adapter.test.ts

```diff
@@ -20,8 +20,11 @@ import * as path from 'path'
 import { DebugClient } from 'vscode-debugadapter-testsupport'
 
 suite('Daffodil Debug Adapter', () => {
-  const DEBUG_ADAPTER = path.resolve('out/adapter/debugAdapter.js')
-  const PROJECT_ROOT = path.resolve('.')
+  const DEBUG_ADAPTER = path.join(
+    __dirname,
+    '../../../out/adapter/debugAdapter.js'
+  )
+  const PROJECT_ROOT = path.join(__dirname, '../../../')
```

Comment:
I think we should still try to resolve these paths instead of using these relative paths, but I won't hold up this PR over it.  It's probably merits a "technical debt" ticket though.

Reply:
It worked okay on mac but seemed to cause issues in windows CI. So some more testing will need to be done on the other OSes, I believe one of the ubuntu CI's ran it fine so maybe just a thing to test out further on windows?